packages/qwik-nx/src/generators/preset/generator.ts

```diff
@@ -0,0 +1,11 @@
+import { Tree } from '@nrwl/devkit';
+import { QwikWorkspacePresetGeneratorSchema } from './schema';
+import applicationGenerator from '../application/generator';
+
+export default async function (
+  tree: Tree,
+  options: QwikWorkspacePresetGeneratorSchema
+) {
+  options.directory = '';
```

Comment:
this makes `directory` property basically useless. Is it intentional?

Reply:
@dmitry-stepanenko 
https://github.com/qwikifiers/qwik-nx/blob/7cb3845cf974f58f3ef8b69919cc2a3358ddeec8/packages/qwik-nx/src/generators/application/utils/normalize-options.ts#L21-L25 

`options.directory = '' ` will use the `names(options.name).fileName;`

There is a bug if I will just reuse the options.directory, it will create `org/apps/qwik-app/qwik-app`.

Reply:
I cannot reproduce this, can you please describe how exactly this comes out?

Reply:
have you tried to run `npx create-nx-workspace org-workspace --preset=qwik-nx`

Reply:
Hm, it's not reproducible for me if I run `npx create-nx-workspace org-workspace --preset=qwik-nx`
<img width="394" alt="image" src="https://user-images.githubusercontent.com/33101123/206906649-8e4049f6-234d-4afb-8c57-ae908e9c8795.png">

But I can actually reproduce by running `npx nx generate qwik-nx:preset abc --no-interactive --dry-run` in an existing repo by replacing actual options with the list of options that is being provided to the `preset` generator by `create-nx-workspace`: 
``` json
{ "_": ["neatspace", "Object]"], "preset": "qwik-nx", "allPrompts": false, "a": false, "skipGit": false, "g": false, "commit": "[object", "/bin/sh": "/Users/dmitriy/.npm/_npx/d2207cf76adb22dc/node_modules/.bin/create-nx-workspace", "name": "neatspace", "appName": "", "nxCloud": false, "packageManager": "npm", "defaultBase": "main", "ci": "", "cli": "nx", "style": "css", "skipInstall": false, "linter": "eslint", "directory": "neatspace", "presetVersion": "undefined", "skipFormat": false, "unitTestRunner": "vitest", "strict": true }
```

Since nx may use/override flags that we're using, it feels like the best what we can do is to modify the schema to have other names there. For example, it can properly generate the app name using this command `npx create-nx-workspace --preset=qwik-nx --qwikAppName=neat-qwik-app`. 

Here's the list of inputs that will be overridden: 
```
      name,
      preset,
      appName,
      style,
      cli,
      nxCloud,
      packageManager,
      defaultBase,
      ci,
```

So I propose to rename the following inputs: 
- name => qwikAppName
- style => qwikAppStyle

and remove the following inputs as they won't be actually used:
- directory
- skipFormat 

Then we can map property names to what `applicationGenerator` needs.

@reemardelarosa @shairez maybe you have any other thoughts on this?

Reply:
@dmitry-stepanenko , let's try this and test.
```
name => qwikAppName ?? options.name
style => qwikAppStyle
```

Reply:
> so the qwikAppName schema change will be just for the preset right?
yes @shairez that's how I think we should do it for now

> Plus, is it something we should maybe discuss with Juri on discord? (maybe it's a feature request for nx, to support regular app names etc)
sounds reasonable, this seems to be a huge limitation

> WDYT about adding a prompt asking if they want the new "standalone" project vs the traditional monorepo project?
unfortunately this is another limitation - prompts does not work with custom presets, so have to discuss it with Juri

Reply:
@shairez  @reemardelarosa I think it's better to ship whatever we can at this point and enhance it later down the road once we're able to

Reply:
@reemardelarosa did you have a chance to implement this yet?

https://github.com/qwikifiers/qwik-nx/pull/10#discussion_r1045711773

if not please let us know and we'll do it

Reply:
Agree with @dmitry-stepanenko, to create separate issue/pr for standalone.
WDYT @shairez ?

Reply:
@reemardelarosa  yes, standalone could be a separate PR
I was talking about the link in my last comment, about implementing the changes you suggested in your last comment

wanted just to ask if you did that work already and didn't push, because I didn't see it in the commits yet and didn't want us to do duplicated work

Thanks!

Reply:
@shairez I did implemented this yet
```
name => qwikAppName ?? options.name
style => qwikAppStyle
```

Reply:
awesome I probably missed it